### PR TITLE
pop quizzes added to mux-functions.md

### PR DESCRIPTION
**Pull request by ayan-biswas0412**
A meaningful title for PR, not like: made changes to xyz.md

Fixes #505 

#### Changes done:
- [x] pop quizzes added to mux-functions.md

#### Screenshots
image

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by ayan-biswas0412** [comment]
@Shivansh2407 review please

File: docs/comb-msi/mux-functions.md [hunk 1]
```diff
@@ -62,3 +62,16 @@ The set $\\{M(1),0,1\\}$ is functionally complete (See section **Functionally Co
...
+
+{:.quiz}
+
+  1. Can one MUX block be said an universal logic block ?
+     1. Yes
+     * No
+  2. Can one DEMUX block be said an universal logic block ?
+     1. Yes
+     * No
+  3. Which of the following option is correct regarding MUX-based gates ?
```
Comment (manjotsidhu) [inline docs/comb-msi/mux-functions.md:74]: Question can be rephrased as something like "Select the MUX-based gates which is/are possible"

Comment (ayan-biswas0412) [inline docs/comb-msi/mux-functions.md:74]:
```suggestion
  3. Select the MUX-based gates which is/are possible
```

**Comment by ayan-biswas0412** [comment]
@manjotsidhu please review I have made the necessary Changes

File: docs/comb-msi/mux-functions.md [hunk 2]
```diff
@@ -62,3 +62,16 @@ The set $\\{M(1),0,1\\}$ is functionally complete (See section **Functionally Co
...
+{:.quiz}
+
+  1. Can one MUX block be said an universal logic block ?
+     1. Yes
+     * No
+  2. Can one DEMUX block be said an universal logic block ?
+     1. Yes
+     * No
+  3. Select the MUX-based gates which is/are possible ?
+     * MUX-based XOR gate is impossible
```
Comment (manjotsidhu) [inline docs/comb-msi/mux-functions.md:75]: is impossible in answers is not required
```suggestion
     * MUX-based XOR gate is impossible
```

**Review by manjotsidhu — CHANGES_REQUESTED** [review]

**Review by manjotsidhu — APPROVED** [review]

**Review by Shivansh2407 — APPROVED** [review]
Nice Work @ayan-biswas0412 🎉 Make sure to squash the commits before finalizing the PR for review. Thanks a lot!

**Comment by ayan-biswas0412** [comment]
> Nice Work @ayan-biswas0412 🎉 Make sure to squash the commits before finalizing the PR for review. Thanks a lot!

yah sure

**Pull request merged**